### PR TITLE
Remove category sync to mailchimp interest groups.

### PR DESCRIPTION
**Pull request by alanpeabody**
Also adds the ISFEATURED merge field.

### CODE REVIEW

**Comment by alanpeabody** [comment]
I left the passing of categories and featured categories to the mailchimp processor in place so when we figure out how/if we want to get those in there we can do so.

**Comment by alanpeabody** [comment]
Pushing this to staging.

File: lib/ello/kinesis_consumer/mailchimp_processor.rb [hunk 1]
```diff
@@ -80,6 +80,7 @@ def merge_fields_for_user(user_record, overrides = {})
           HAS_BIO: bool(user_record['has_bio']),
           HAS_LINKS: bool(user_record['has_links']),
           LOCATION: user_record['location'],
+          ISFEATURED: bool(user_record['is_featured']),
```
Comment (jayzes) [inline lib/ello/kinesis_consumer/mailchimp_processor.rb]: Is this serialized off of the `is_featured` column in the database or the presence of any featured categories?

Comment (jayzes) [inline lib/ello/kinesis_consumer/mailchimp_processor.rb]: Nevermind, just saw your mothership PR. Carry on.

**Pull request merged**